packages/editor/src/components/char-count/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { _x } from '@wordpress/i18n';
+import { count as charCount } from '@wordpress/wordcount';
+
+function CharCount( { content } ) {
+	/*
+	 * Only count the number of characters (no spaces).
+	 * Do not translate into your own language.
```

Comment:
Might be good to list the two options here: `characters_excluding_spaces` and `characters_including_spaces`.

Reply:
Done.